src/connection.rs

```diff
@@ -0,0 +1,143 @@
+use crate::{any_glasses, ARGlasses, Error, GlassesEvent};
+use nalgebra::{UnitQuaternion, Vector3};
+use std::ops::Deref;
+
+type Result<T> = std::result::Result<T, Error>;
+
+mod c_f {
```

Comment:
The constants can go into the module directly, no need for a specific namespace.

Or they could go into the struct impl.